src/App.tsx

```diff
@@ -1,156 +1,89 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import { createTodo, getTodos } from './api/todos';
+import { Footer } from './components/Footer/Footer';
+import { Notification } from './components/Notification/Notification';
 
-const USER_ID = 0;
+const USER_ID = 11085;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<'all' | 'active' | 'completed'>('all');
```

Comment:
```suggestion
  const [filter, setFilter] = useState<'all' | 'active' | 'completed'>('all');
```
you write `'all' | 'active' | 'completed'` a lot of times - create type for it and export everywhere you need